helpers/build_packages.sh

```diff
@@ -301,23 +304,23 @@ if [ "$BUILDGG" = "1" ]; then
 
     if grep -qs "^- gstreamer1.0-droid" "$pattern_lookup" ||
        grep -qs "^Requires: gstreamer1.0-droid" "$metapackage_lookup"; then
-        droidmedia_version=$(git --git-dir external/droidmedia/.git describe --tags 2>/dev/null | sed -r "s/\-/\+/g")
+        pkg=droidmedia
+        cd external/$pkg || die "Could not change directory to external/$pkg"
+        droidmedia_version=$(get_package_version "$pkg")
         if [ -z "$droidmedia_version" ]; then
-            # in case of shallow clone:
-            droidmedia_version=999.99999999.99
+            # Could not obtain version, function call will have shown the error
```

Comment:
get_package_version() already exits on failure.

Reply:
`get_package_version()` is called in a sub-shell, thus exit just exits that sub-shell.